definitions/procedures/pulp/remove.rb

```diff
@@ -163,12 +170,41 @@ def drop_migrations
     end
     # rubocop:enable Metrics/BlockLength
 
-    def delete_pulp_data(rm_cmds)
-      with_spinner('Deleting pulp2 data directories') do |spinner|
-        rm_cmds.each do |cmd|
-          execute!(cmd)
+    def delete_pulp_data
+      non_mountpoints = data_dir_removal_cmds(deletable_pulp_dirs)
+      mountpoints = pulp_data_dirs_mountpoints
+      with_spinner('') do |spinner|
+        if non_mountpoints.any?
+          spinner.update('Deleting pulp2 data directories.')
+          non_mountpoints.each do |cmd|
+            execute!(cmd)
+          end
+          msg_for_del_non_moutpoints(mountpoints, spinner)
+        end
+        if mountpoints.any?
+          msg_for_del_mountpoints(mountpoints, spinner)
         end
-        spinner.update('Done deleting pulp2 data directories')
+      end
+    end
+
+    def msg_for_del_non_moutpoints(mountpoints, spinner)
+      if mountpoints.empty?
+        spinner.update('Done deleting all pulp2 data directories.')
+      else
+        spinner.update("Deleted only: #{deletable_pulp_dirs.join("\n")}")
```

Comment:
I think `only` is a bit redundant and can be left out:
```suggestion
        spinner.update("Deleted: #{deletable_pulp_dirs.join("\n")}")
```